man/fi_usnic.7.md

```diff
@@ -76,10 +76,118 @@ libfabric interface.
 
 These extensions are available via the "fi_ext_usnic.h" header file.
 
-The following is an example of how to utilize the usnic "fabric getinfo"
-extension, which returns IP and SR-IOV information about a usNIC
-interface obtained from the [`fi_getinfo`(3)](fi_getinfo.3.html)
-function.
+## Fabric Extension: getinfo
+
+The "fabric getinfo" extension was introduced in Libfabric release v1.0.0 and
+can be used to retrieve IP and SR-IOV information about a usNIC device obtained
+from the [`fi_getinfo`(3)](fi_getinfo.3.html) function.
+
+The "fabric getinfo" extension is obtained by calling `fi_open_ops` and
+requesting `FI_USNIC_FABRIC_OPS_1` to get the usNIC fabric extension
+operations.  The `getinfo` function accepts a version parameter that can be
+used to select different versions of the extension. Version 2 is the current
+version and should be used instead of version 1 where possible. The information
+returned by the "fabric getinfo" extension is accessible through a
+`fi_usnic_info` struct that uses a version tagged union. The accessed union
+member must correspond with the requested version. It is recommended that
+applications explicitly request a version rather than using the header provided
+`FI_EXT_USNIC_INFO_VERSION`.
```

Comment:
Re-orient this whole entry to focus on version 2.  At the end of it, make a short mention that there's a version 1, but its use is discouraged, and it may disappear in a future release.